### ISSUE TITLE
Publish releases

### ISSUE THREAD

**Issue by ronaldtse** [tamatebako/libdwarfs#59]
Make this code available for others, e.g. via CMake.

**Comment by maxirmx** [comment]
This version has hardcoded mount point name '/__tebako_memfs__' and that is probably not very good for published release.

**Comment by ronaldtse** [comment]
Good point! This makes me wonder if we can support mixed mount points (as we move towards an overlay-like mount system)...
